scripts/run-pelorus-e2e-tests

```diff
@@ -189,6 +189,25 @@ if [ -z "${ci_filename}" ]; then
     ci_filename="values.yaml"
 fi
 
+if ! type oc &> /dev/null; then
+    echo "OpenShift CLI is necessary to run the script."
+    exit 1
+fi
+
+if ! type helm &> /dev/null; then
+    echo "helm is necessary to run the script."
+    exit 1
+fi
+
+# if ! oc whoami &> /dev/null; then
+#     echo "You must be logged in to your cluster to run the script."
+#     exit 1
+# fi
+
+# if [ "$(oc whoami)" != "kube:admin" ]; then
```

Comment:
I added this commented commands because of what I read in the documentation. It is really necessary to check username?